app/portainer/feature-flags/enums.ts

```diff
@@ -30,4 +30,5 @@ export enum FeatureId {
   STACK_WEBHOOK = 'stack-webhook',
   CONTAINER_WEBHOOK = 'container-webhook',
   POD_SECURITY_POLICY_CONSTRAINT = 'pod-security-policy-constraint',
+  HIDE_DOCKER_HUB_ANONYMOUS = 'hide-docker-hub-anonymous',
```

Comment:
is this feature id is synced with PO? should work, so that `https://www.portainer.io/business-upsell?from=hide-docker-hub-anonymous` will point to something

Reply:
just saw PO's comment. good